jsconfig.json

```
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+      "baseUrl": ".",
+      "paths": {
+        "@/components/*": ["components/*"],
+        "@/pages/*":["pages/*"],
+        "@/assets/*":["../assets/*"],
```

Do not use relative paths here.

```suggestion
        "@/assets/*":["public/assets/*"],
```